infra/datalake/processing_stack.py

```diff
@@ -0,0 +1,309 @@
+            self,
+            "batch-launch-template",
+            launch_template_name="datalake-batch-launch-template",
+            launch_template_data={
+                "userData": core.Fn.base64(batch_launch_template_data.strip()),
+            },
+        )
+
+        if ENV == "prod":
+            instance_types = [
+                aws_ec2.InstanceType("c5.xlarge"),
+                aws_ec2.InstanceType("c5.2xlarge"),
+                aws_ec2.InstanceType("c5.4xlarge"),
+                aws_ec2.InstanceType("c5.9xlarge"),
+            ]
+        else:
+            instance_types = [
+                aws_ec2.InstanceType("m5.large"),
+                aws_ec2.InstanceType("m5.xlarge"),
+            ]
+
+        batch_compute_environment = aws_batch.ComputeEnvironment(
+            self,
+            "compute-environment",
+            compute_resources=aws_batch.ComputeResources(
+                vpc=vpc,
+                # vpc_subnets=vpc.select_subnets(subnet_group_name="ecs-cluster"),  # TODO
+                minv_cpus=0,
+                desiredv_cpus=0,
+                maxv_cpus=1000,
+                instance_types=instance_types,
+                instance_role=batch_instance_profile.instance_profile_name,
+                allocation_strategy=aws_batch.AllocationStrategy("BEST_FIT_PROGRESSIVE"),
+                launch_template=aws_batch.LaunchTemplateSpecification(
+                    launch_template_name=batch_launch_template.launch_template_name
+                ),
+            ),
+            service_role=batch_service_role,
+        )
+
+        batch_job_queue = aws_batch.JobQueue(
+            self,
+            "dataset-version-creation-queue",
+            compute_environments=[
+                aws_batch.JobQueueComputeEnvironment(
+                    compute_environment=batch_compute_environment, order=10
+                ),
+            ],
+            priority=10,
+        )
+
+        # BUNDLING AND TASKS BUILDING
+        step_tasks = {}
+        for task in creation_tasks:
+
+            # lambda functions
+            if creation_tasks[task]["type"] == "lambda":
+
+                lambda_function = aws_lambda.Function(
+                    self,
+                    f"{task}-function",
+                    handler=f"processing.{task}.task.lambda_handler",
+                    runtime=aws_lambda.Runtime.PYTHON_3_6,
+                    code=aws_lambda.Code.from_asset(
+                        path="..",
+                        bundling=core.BundlingOptions(
+                            # pylint:disable=no-member
+                            image=aws_lambda.Runtime.PYTHON_3_6.bundling_docker_image,
+                            command=["backend/processing/bundle.bash", f"{task}"],
+                        ),
+                    ),
+                )
+
+                step_tasks[task] = aws_stepfunctions_tasks.LambdaInvoke(
+                    self,
+                    f"{task}",
```

Comment:
This could be just `task,`

Reply:
Fixed in d9d1f2b817315e64a8669f7f3680019e23518083